dbms/programs/client/Client.cpp

```diff
@@ -462,6 +462,11 @@ class Client : public Poco::Util::Application
             if (rl_initialize())
                 throw Exception("Cannot initialize readline", ErrorCodes::CANNOT_READLINE);
 
+#if RL_VERSION_MAJOR >= 7
```

Comment:
Ok but see `-Wundef` when readline is not used.

Reply:
But it's already guarded by `USE_READLINE`

Reply:
Ok.